src/python/pants/backend/python/packaging/pyoxidizer/rules.py

```diff
@@ -90,13 +94,9 @@ async def package_pyoxidizer_binary(
     runner_script: PyoxidizerRunnerScript,
     bash: BashBinary,
 ) -> BuiltPackage:
-    direct_deps, pyoxidizer_pex = await MultiGet(
```

Comment:
I made building the PEX lazier.

Reply:
Nit: since it's not possible for this `@rule` to succeed without a matching target type, it's not clear that this will end up skipping any work in anything but the error case (and only the first time). I see though that you wanted to speed up the failing test though.

One thing that _would_ probably make it faster in the successful case would be to include the building of the pyoxidizer binary in the multiget for the `built_packages`. That's unfortunately not something that we have syntax for though.

Reply:
Yeah, this only speeds up the error case, indeed.